data/discovery/field/statistical-geography.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: integer
```

Comment:
Make this a string as we're unlikely to do any maths on it and it's an unnatural constraint for an identifier.